internal/user/repo_interface.go

```diff
@@ -0,0 +1,9 @@
+package user
+
+import repomodel "github.com/porky256/mock-interview-tbr/internal/models/repo"
+
```

Comment:
as I said interfaces should be created where they are used. There is no need to put them in different file, unless we have 100500 methods to implement

Reply:
it's make no sense because then we should put interface to every possible DB implementation we will have

Reply:
We can use one interface placed in service layer

Reply:
because I guess service and the repo will both implement one interface

Reply:
it makes more convenient to generate smth (for example mocks) from separated file with necessary interfaces/functions